DESCRIPTION

```diff
@@ -22,7 +22,10 @@ Imports:
     msm,
     methods,
     uuid,
-    graphics
+    graphics,
+    filelock,
+    parallel,
+    doParallel
```

Comment:
pytanie czy dajemy to tu czy do suggests? bo chyba nie chcemy żeby każdy musiał to instalować

Reply:
oki, przegapiłem zupełnie istnienie czegoś takiego jak to suggests, przerzuciłem to tam